templates/mwe.md

```diff
@@ -1,17 +1,5 @@
 <div class="alert alert-info">
 <p>
-We are currently updating the Lean community website to describe working with Lean 4,
-but most of the information you will find here today still describes Lean 3.
-</p>
-<p>
-Pull requests updating this page for Lean 4 are very welcome.
-There is a link at the bottom of this page.
-</p>
-<p>
-Please visit <a href="https://leanprover.zulipchat.com">the leanprover zulip</a>
-and ask for whatever help you need during this transitional period!
-</p>
-<p>
```

Comment:
I think you need to remove the entire div starting on line 1.

Reply:
I thought it might be useful to still have the link to the archived lean3 page for some time, but I can remove it.

Reply:
No, we want people to forget about Lean 3. Please remove that div.